src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.stories.tsx

```diff
@@ -24,7 +25,17 @@ type AdvancedSearchResultDetailsListStory = ComponentStory<
 >;
 
 const Template: AdvancedSearchResultDetailsListStory = (args) => {
-    return <AdvancedSearchResultDetailsList {...args} />;
+    const authenticationParameters = useAuthParams();
```

Comment:
why do we need this for a mock story?

Reply:
That's my mistake, no need for it. removing...